### ISSUE TITLE
gitea dump doesn't backup relocated data (lfs, attachments, etc)

### ISSUE THREAD

**Issue by snoopydo** [go-gitea/gitea#12828]
- Gitea version (or commit ref): 1.12.4
- Git version: 2.28.0.windows.1
- Operating system: Windows 10 & Windows Server 2012
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [X] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [X] Not relevant
- Log gist:

## Description

When running ``gitea dump`` the zip archive doesn't actually contain any repository or lfs data.
gitea is installed into c:\gitea with standard custom, data & log sub folders, however in my app.ini i have specified
````
[repository]
ROOT = F:/Gitea/Data/repositories

[attachment]
PATH = F:/Gitea/Data/attachments

[server]
LFS_CONTENT_PATH = F:/Gitea/data/lfs
````

update 1: gitea and git versions.
update 2: changed title, repositories are backed up.

**Comment by jolheiser** [comment]
Did you pass the config flag to the dump command?

**Comment by snoopydo** [comment]
Yes.  I tried both methods from c:\gitea,  ``gitea dump`` and ``gitea dump -config c:\Gitea\custom\conf\app.ini``

**Comment by jolheiser** [comment]
Try with the config flag before the `dump` sub-command

**Comment by 6543** [comment]
@snoopydo -c or --config

**Comment by snoopydo** [comment]
I ran ``gitea --config c:\Gitea\custom\conf\app.ini dump --skip-log``

after a bit more digging, in the ``gitea-dump .zip`` there is ``gitea-repo.zip`` which appeared to be an empty/corrupt file which has the repositories in it(after using winzip to look at it, windows native zip view shows nothing).  However both the attachments and lfs data are still not included.

**Comment by snoopydo** [comment]
Observations after some more checking.
1. repositories are backed up, though archive appears empty/broken on windows via the native archive viewer app
2. data folders that are moved out of the install location (via config settings) are missed from the backup
3. there can be two copies of the repositories in the back, 1 via the zip file and the 2nd via copy of the data folder.

**Comment by lunny** [comment]
I think we have resolved this on #12813
